src/partials/footer.html

```diff
@@ -0,0 +1,20 @@
+<footer class="footer">
+  <div class="footer__container">
```

Comment:
we missing helpfull class named "container"

Reply:
Deleted